wpilib/wpilib/_impl/utils.py

```diff
@@ -206,3 +207,77 @@ def reset_wpilib():
                 
     import hal_impl.functions
     hal_impl.functions.reset_hal()
+
+string_types = (type(b''), type(u''))
+def deprecated(reason):
```

Comment:
This whole decorator doesn't even seem to be useful. You're using this, _and_ also warning a DeprecationWarning in the single place it's being used.

Reply:
Not gonna lie I copy/pasted this and thought it would only get called when the method initializes. Lemme double check to make sure it's doing what I think